include/grpcpp/ext/otel_plugin.h

```diff
@@ -67,8 +53,27 @@ OpenTelemetryServerCallRcvdTotalCompressedMessageSizeInstrumentName();
 /// grpc.server.call.rcvd_total_compressed_message_size
```

Comment:
Probably no need to list the metrics in the comment now that the constants are in the header file directly.

Reply:
done